### DISCUSSION TITLE
Integration of ICS Layer into Navigator

### DISCUSSION THREAD

**Discussion by YannickZ5** [Security-Onion-Solutions/securityonion#9120] [Unsupported Versions] [answered]
Hello everyone,

we are running SO in version 2.3.180 in an ICS environment as an airgaped implementation. We have PLCs and other OT-/IoT-Devices which are sending syslog messages to SO and we have also some custom playbook rules in place for this kind of logs which have an attck reference to ICS IDs. Is it possible to import the ICS Layer in the Navigator an make a mapping from playbook as it already is with the enterprise layer?

Thank you for your help!

**Comment by defensivedepth** [comment, marked as answer]
I have created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/9144
